README.md

```diff
@@ -133,7 +128,7 @@ ngrok http 8443 -host-header="localhost:8443"
 
 ## Formatting and Linting
 
-The code in this repository is formatted using `prettier`. If you have prettier setup in your code editor it should work out of the box; otherwise you can use the `pnpm lintfix` script to format and fix lint errors in your code. Checks are run to lint your code and validate the formatting on git precommit using [husky](https://github.com/typicode/husky).
+The code in this repository is formatted using `prettier`. If you have prettier setup in your code editor it should work out of the box; otherwise you can use the `pnpm lint:fix` and `pnpm format:fix` scripts to format and fix lint errors in your code. Checks are run to lint your code and validate the formatting on git precommit using [husky](https://github.com/typicode/husky).
```

Comment:
I wonder why we split `format` and `lint` in this way? Could we just have a single `lint:fix` script that runs the formatting fix?

Reply:
Mentally I associated `lint` with ESLint and `format` with Prettier. While they do overlap in the formatting department, aren't their concerns slightly different?

Reply:
They are technically different, but we don't make the distinction in our Python code (it's all just one big pre-commit run with everything, formatting and linting included) and I think I do commonly see prettier grouped under linting :shrug: 

Is there ever a situation in which you would want to run one and not the other? It seems to me that both should always run whenever one or the other is run (which to me is the most compelling reason to make them a single script, despite there being a technical difference between the two).

Reply:
That makes sense, one less command to remember. Combined in b4a001c.

Reply:
Nice! That is much nicer to work with :tada: